Units/review-needed.r/simple.js.t/input.js

```diff
@@ -40,8 +39,6 @@ function validFunctionThree(a,b) {
 
 var my_global_var1 = 33;
 
-function extra.validFunctionFour(a,b) {}
```

Comment:
Why do we have to remove this?

A. the limitation of this refactor. we will support this syntax in the future.
For A, you can move this fragment to ".b" test case.

B. the parser should not support this syntax for some reason.
For B, could you write the reason or background to the commit log?
This is a "refactor" commit. For such a commit, we expect all test cases for the parser should be passed without modification.

Reply:
This is invalid JavaScript.  I will create a separate commit for it.